pkg/vgmanager/vgmanager_controller.go

```diff
@@ -381,3 +382,14 @@ func (r *VGReconciler) getNewNodeStatus(status *lvmv1alpha1.VGStatus) *lvmv1alph
 	setStatus(status, vgNodeStatus)
 	return vgNodeStatus
 }
+
+func (r *VGReconciler) matchesMyNode(ctx context.Context, selector *corev1.NodeSelector) (bool, error) {
```

Comment:
- maybe we'll not be using this function again and considering it's quite small, how about in-lining it in the call site itself?

Reply:
I think that is best left to the compiler.

Reply:
ah! got it.

Reply:
I also agree with Leela here. We can remove the `matchesMyNode` function.

Reply:
I prefer having functions for readability.